MAGECLOUD-4056: [Cloud Docker] Static files not available if these are created on build stage (#5157)
magento/devdocs: guides/v2.2/cloud/docker/docker-config.md

    ```bash
    docker-compose run deploy magento-command cache:clean
    ```

> 1. _Optional_: Restart services. In cases when static content didn't synchronize with all images after generation on build phase.

Reviewer: ```suggestion
1. _Optional_: Restart services if the static content does not synchronize with all images after generation on build phase.
```
